content/operate/rc/accounts/user-profile.md

```diff
@@ -56,6 +56,10 @@ When your user account is authorized to manage multiple Redis Cloud accounts, ea
 
 To switch accounts, select the desired account from the list shown in the Profile control.
 
+{{< note >}}
+To create another Redis Cloud account associated with the same email address, you need to [sign up for Redis Cloud]({{< relref "/operate/rc/rc-quickstart" >}}) again with a sub-address of the email address. For many email providers, you can do this by adding `+` and any string to the end of your username. For example, if your address is `example@example.com`, enter `example+account2@example.com`.
```

Comment:
@cmilesb One thought: should we maybe call out that "sub-addressing" it is also known as "plus addressing"? The latter comes more intuitive to me. Other than that look good. Thanks!

Reply:
```suggestion
To create another Redis Cloud account associated with the same email address, you need to [sign up for Redis Cloud]({{< relref "/operate/rc/rc-quickstart" >}}) again with a plus address of the email address. For many email providers, you can do this by adding `+` and any string to the end of your username. For example, if your address is `example@example.com`, enter `example+account2@example.com`.
```